src/Hl7.Fhir.Specification/Specification/StructureDefinitionSummaryProvider.cs

```diff
@@ -233,10 +233,11 @@ public XmlRepresentation Representation
         private static bool isResource(ElementDefinition defn) => defn.Type.Count == 1 && 
             (defn.Type[0].Code == "Resource" || defn.Type[0].Code == "DomainResource");
 
+        public string DefaultTypeName => _definition.GetExtensionValue<FhirString>("http://hl7.org/fhir/StructureDefinition/elementdefinition-defaulttype")?.Value;
```

Comment:
Can we use this extension? It is not defined here: https://www.hl7.org/fhir/elementdefinition-extras.html

Reply:
Discussed the issue on Zulip: https://chat.fhir.org/#narrow/stream/179177-conformance/topic/CCDA.20Logical.20Model.20Type.20Representation

Created https://jira.hl7.org/browse/FHIR-25440 to add it officially to the list.